source/setup.py

```diff
@@ -92,6 +57,8 @@ def initialize_options(self):
 		self.enable_uiAccess = False
 
 	def run(self):
+		with open(manifestTemplateFilePath, "r") as manifestTemplateFile:

```

Comment:
It would be better to specify explicit encoding here - newer versions of Python tent to show a warning when `open` is used without explicit encoding
```suggestion
		with open(manifestTemplateFilePath, "r", encoding="utf-8") as manifestTemplateFile:
```